react/src/pages/datasets/DatasetInfoDialog.tsx

```diff
@@ -66,7 +51,8 @@ export default function DatasetInfoDialog({ dataset_id, open, onClose }: DialogP
     const [dataset, setDataset] = useState<Dataset>();
     const [analyses, setAnalyses] = useState<Analysis[]>([]);
     const [sample, setSample] = useState<Sample>();
-
+    //for updating the dialog content when re-open the dialog
+    const [num, reRender] = useState(0);
```

Comment:
You don't need to force a re-render. Change this dialog to accept the dataset object as a prop, and add it to the `useEffect` dependencies below, same as in ParticipantInfoDialog. `useEffect` is not fetching the dataset again because `dataset_id` does not change.